ProofOfConcept/src/main/java/edu/mum/cs544/wind/domain/Person.java

```diff
@@ -54,7 +58,7 @@ public long getId() {
         return id;
     }
 
-    private void setId(long id) {
+    public void setId(long id) {
```

Comment:
@rafaelmum and why do you change this setId() to public, we shouldn't be able to change the id once it's generated, that's why I put it private.